contracts/nft/NFTInterestFacet.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+// Contracts
+import { NFTFacet } from "./NFTFacet.sol";
+
+// Libraries
+import { NFTLib } from "./libraries/NFTLib.sol";
+import { NumbersLib } from "../shared/libraries/NumbersLib.sol";
+
+contract NFTInterestFacet is NFTFacet {
```

Comment:
we should add a function here that calls the TToken to claim the interest on behalf of the caller. as the claim interest is protected to only be callable from the Diamond 

it should calculate the percentage to use and then call the function you made in `TToken_V3` `claimAlphaInterest` that forwards the callers address and the calculated percentage.

Reply:
This is what I was asking about last week. Will add it in